config/grunt/custom-tasks/init-rc-release.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+/**
+ * helper function
```

Comment:
Please improve the description of this piece of code.

Reply:
changed it to: 
```
'use strict';

/**
 * helper function adds a function to a string to determan is that sctring in the list specified list 
 * @param {array} 
 * @returns {bool}
 */

if (!String.prototype.isInList) {
	String.prototype.isInList = function() {
	   let value = this.valueOf();
	   for (let i = 0, l = arguments.length; i < l; i += 1) {
		  if (arguments[i] === value) return true;
	   }
	   return false;
	}
 }

/**
 * Grunt task Initializes the configuration needed to do an RC release..
 *
 * @param {Object} grunt The grunt helper object.
 *
 * @returns {void}
 */
module.exports = function( grunt ) {
```